### PR TITLE
fix(be): Remove validation:required from policy description

### PR DESCRIPTION
**Pull request by clickboo**
### Description

change me!

### User-facing documentation

- [ ] CHANGELOG is updated **OR** update is not needed
- [ ] [documentation PR](https://spaces.redhat.com/display/StackRox/Submitting+a+User+Documentation+Pull+Request) is created and is linked above **OR** is not needed

### Testing and quality

- [ ] the change is production ready: the change is GA or otherwise the functionality is gated by a feature flag
- [ ] CI results are inspected

#### Automated testing

- [ ] added unit tests
- [ ] added e2e tests
- [ ] added regression tests
- [ ] added compatibility tests
- [ ] modified existing tests

#### How I validated my change

change me!

### CODE REVIEW

File: config-controller/api/v1alpha1/policy_types.go [hunk 1]
```diff
@@ -40,7 +40,6 @@ type SecurityPolicySpec struct {
 	// +kubebuilder:validation:Pattern=`^[^\n\r\$]{5,128}$`
 	// PolicyName is the name of the policy as it appears in the API and UI.  Note that changing this value will rename the policy as stored in the database.  This field must be unique.
 	PolicyName string `json:"policyName"`
-	// +kubebuilder:validation:Required
```
Comment (vjwilson) [inline config-controller/api/v1alpha1/policy_types.go:43]: Isn't this requiring `policyName` ?

It was `description` that gave me the error.

Comment (clickboo) [reply config-controller/api/v1alpha1/policy_types.go:43]: Line 39 is for policyName. This one is for description. Comments before the field apply to the field.

Comment (vjwilson) [reply config-controller/api/v1alpha1/policy_types.go:43]: Ugh, preceding comments as decorators?

I am asleep and this is a nightmare, isn't it?

Comment (clickboo) [reply config-controller/api/v1alpha1/policy_types.go:43]: The nightmare will be over once you hit the approve button.

**Review by vjwilson — APPROVED** [review]
/lgtm

**Pull request merged**